.changeset/yellow-spies-design.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ts-r/core": minor
```

Comment:
Can we change this to be patch? I'd like to keep the version numbers down

Reply:
Sure